Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -2586,14 +2586,12 @@ def handle_outgoing_issue_closure(remote_args):
     current_remote_status = remote_args.data.get('status') if remote_args.data else None
     # force closing remote incident only if:
     #   The XSOAR incident is closed
-    #   and the closingUserId was changed
     #   and the remote incident isn't already closed
     if remote_args.inc_status == 2 and \
-       update_args.get('closingUserId') and \
        current_remote_status not in XDR_RESOLVED_STATUS_TO_XSOAR:
 
-        update_args['resolve_comment'] = update_args.get('closeNotes', '')
-        update_args['status'] = XSOAR_RESOLVED_STATUS.get(update_args.get('closeReason', 'Other'))
+        update_args['resolve_comment'] = update_args.get('closeNotes') or update_args.get('resolve_comment', '')
```

Comment:
Is there a case that `update_args` already contained `resolve_comment` or `status` ?

Reply:
Yes.
This is the args of an incident closed with a comment.
```
    {
        "data": {
            "manual_severity": "high",
            "resolve_comment": "testing bla",
            "status": "new"
        },
        "delta": {
            "resolve_comment": "testing bla"
        },
        "entries": None,
        "incidentChanged": True,
        "remoteId": "59",
        "status": 2,
        "integration_context_brand": "PaloAltoNetworksXDR",
        "integration_name": "Cortex XDR - IR"
    }
```